wds-required-plugins.php

```diff
@@ -112,11 +112,6 @@ public function required_text_markup() {
 	 * @return array
 	 */
 	public function filter_plugin_links( $actions = array(), $plugin ) {
-		// Remove edit link for all plugins
-		if ( array_key_exists( 'edit', $actions ) ) {
-			unset( $actions['edit'] );
-		}
-
```

Comment:
If all we want to do is force a requirement of active plugins then I think removing the edit button is out of that scope